lib/cluster.js

```diff
@@ -273,17 +274,18 @@ function masterInit() {
     if (hasDebugArg)
       return;
 
-    process.on('internalMessage', function(message) {
+    process.on('internalMessage', function internalMessageHandler(message) {
       if (message.cmd !== 'NODE_DEBUG_ENABLED') return;
       var key;
       for (key in cluster.workers) {
         var worker = cluster.workers[key];
         if (worker.state === 'online' || worker.state === 'listening') {
           process._debugProcess(worker.process.pid);
         } else {
-          worker.once('online', function() {
+          const onlineCallbackFunction = () => {
```

Comment:
Why move this out to an arrow function when you could simply name the existing function?